README.md

```diff
@@ -2,712 +2,46 @@
-   * If you have been debugging your app, crash reporting won't work with a debugger attached because the presence of a debugger makes crash reporting impossible.
-   * The user needs to upload the symbols that match the UUID of the build that triggered the crash.
-   * Make sure your device is online.
-   * Check if the App Secret used to start the SDK matches the App Secret in Mobile Center portal.
-   * Don't use any other SDK that provides Crash Reporting functionality.
-
-* What permissions or entitlements are required for the SDK?   
-  
-  Mobile Center SDK requires no permissions to be set in your app.
-  
-* The Alert that prompts users for an update doesn't contain strings, but just the keys for them?
-  This means that the `MobileCenterDistributeResources.bundle` wasn't added to the project. Make sure you have drag'n'dropped the file into your xcode project, and it appears in your app target's `Copy Bundle Resources` build phase. The later should be the case if you have added the file through drag'n'drop – Xcode does it automatically for you. If the file is missing from the build phase, add it so it get's compiled into your app's bundle. 
-  
-* Engage with other MobileCenter users and developers on [StackOverflow](http://stackoverflow.com/questions/tagged/mobile-center).
-
-## 9. Contributing
-
-We're looking forward to your contributions via pull requests.
-
-### 9.1 Code of Conduct
+You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact opencode@microsoft.com with any additional questions or comments.
+## 3. Contact
 
-### 9.2 Contributor License
+If you have further questions, want to provide feedback or you are running into issues, feel free to [open a Github issue](https://github.com/Microsoft/mobile-center-sdk-ios/issues).
 
-You must sign a [Contributor License Agreement](https://cla.microsoft.com/) before submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to submit a request via the [form](https://cla.microsoft.com/) and then electronically sign the CLA when you receive the email containing the link to the document. You need to sign the CLA only once to cover submission to any Microsoft OSS project. 
+### 3.1 E-Mail
+Don't hesitate to get in touch via email to [mobilecentersdk@microsoft.com](mailto:mobilecentersdk@microsoft.com).
```

Comment:
Do we want to get feedback via this email?  Would it be better to omit it & just direct to Intercom instead?